diff --git a/src/Gui/Assistant.cpp b/src/Gui/Assistant.cpp
index f903c7a2..9ee53c9f 100644
--- a/src/Gui/Assistant.cpp
+++ b/src/Gui/Assistant.cpp
@@ -90,7 +90,7 @@ bool Assistant::startAssistant()
 #elif defined(Q_OS_MAC)
         QString app = QCoreApplication::applicationDirPath() + QDir::separator();
 #else
-        QString app = QLibraryInfo::location(QLibraryInfo::BinariesPath) + QDir::separator();
+        QString app = /*QLibraryInfo::location(QLibraryInfo::BinariesPath)*/ QString::fromUtf8("/app/bin") + QDir::separator();
 #endif
         app += QLatin1String("assistant");
 
